src/test/java/org/assertj/core/api/recursive/comparison/iterables/RecursiveIterableComparisonAssert_contains_Test.java

```diff
@@ -0,0 +1,60 @@
+package org.assertj.core.api.recursive.comparison.iterables;
+
+import org.assertj.core.internal.objects.data.Person;
+import org.assertj.core.internal.objects.data.PersonDto;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+
+public class RecursiveIterableComparisonAssert_contains_Test {
```

Comment:
I love tests following the GIVEN WHEN THEN pattern, let's do that for all tests ;-)